ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -173,6 +178,43 @@ class RootViewController: UIViewController, UITableViewDataSource, UITableViewDe
         return tableView
     }()
 
+    private(set) lazy var searchProgressSpinnerSwitchView: UIView = {
+        let itemRow = UIStackView()
+        itemRow.axis = .horizontal
+        itemRow.distribution = .equalCentering
+        itemRow.alignment = .leading
+        itemRow.isLayoutMarginsRelativeArrangement = true
+        itemRow.layoutMargins = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
```

Comment:
Can you set the left and right insets to 15 to match the icons of the table view cells below?